internal/main.go

```diff
@@ -50,6 +53,11 @@ func main() {
 		return
 	}
 
+	if flags.version {
+		fmt.Printf("%s\n", version.String)
```

Comment:
Following the Ignition example, I put `ct` inside `version.String`.  I can move it though.